docs/tpd_requirements.rst

```diff
@@ -0,0 +1,164 @@
+
+*******************
+Repository Template
+*******************
+
+We have standardized what a repository's structure should look like to better assist our downloader system and provide essential information to the Red portal.
+
+The main repository should contain at a minimum:
+ 
+ - A readme file (can be either .MD or .rst)
+ - A info.json file (`examples <tpd_requirements.html#info-jsons>`_)
+ - One folder for each cog package in the repository
+ - A license (Not required, but recommended)
+
+For a simple example of what this might look like when finished, take a look at our example template.
+
+README
+======
+The readme file can be either an .MD or a .rst. However, it should contain, at a minimum the following: 
+
+  **Repository Name** - Examples: Snuffy's Cogs, Bobo's Plugins, Cat-Toys, etc
+
+  **Installation Instructions** - Tell the user how to download your repo and access your cogs. Any additional hurdles to using your 
+  creations should be added here as well.
+
+  **Requirements** - Any third party libraries or other requirements you feel necessary for the use of your cogs.
+
+  **Credits** - You must credit anyone whose code you might have used or whom you received a substantial amount of help from.
+
+  **Contact** - How to contact you about bugs, issues, or suggestions. 
+
+  **License** - This is not required, but we do recommend having one.
+
+
+Info Jsons
+==========
+There are two info.json files. One that goes into each cog folder and one at the main level of your repository. Each have similar keys, but serve a different purpose. The following keys are required:
+
+**Repository Level**
+
+ - author
+ - install_msg
+ - name
+ - short
+ - requirements
+ - description
+ - tags
+
+**Cog Level**
+
+ - author
+ - install_msg
+ - name
+ - short
+ - requirements
+ - description
+ - tags
+ - min_python_version
+ - permissions (Senior Only)
+
+You may include as many optional keys as you wish. For more information these keys and their meaning, `read about it here <framework_downloader.html#info-json>`_
+
+.. note::
+
+    If your cog comes bundled with data, or creates any folder or files (outside of config), then this must be disclosed in your 
+    installation message.
+
+Cog Package Folder
+^^^^^^^^^^^^^^^^^^
+The cog folder has the following requirements:
+
+- The folder name, must match the cog.py name. It should be in all lowercase letters
+- The folder should contain an info.json file. Read above for more details on the specific keys needed
+- An __init__.py for the cog
+- The .py file for the cog. (in all lowercase letters)
```

Comment:
> The .py file for the cog. (in all lowercase letters)

This line can be removed. There's nothing in the downloader's design requiring this to be how a cog folder is structured, just that a setup file is discoverable in the folder, and that the folder is structured as a module.